samples/canopen/canopen_sample/main.cpp

```diff
@@ -46,35 +46,6 @@ void canInterrupt(IO::CANMessage& message, void* priv) {
         queue->append(message);
 }
 
-///////////////////////////////////////////////////////////////////////////////
```

Comment:
This looks like it would break to me, but it should be fixed with your macro stuff, so I'm not worried about it